src/js/themes/base.js

```diff
@@ -1270,6 +1270,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // any box props
         align: 'end',
       },
+      // pad: undefined,
```

Comment:
Do we want to make the Grommet default what we want in HPE Theme? I believe Eric was running into similar spacing issues when using PageHeader in Grommet Designer -- so maybe it's a good default for all.

Reply:
Done.